README.md

```diff
@@ -123,12 +123,13 @@ chmod 400 ./tests/utils/keyfile && \
 VAULT_DB_BACKEND=MEMORY node vaultd.js
 ```
 
-In a new shell, start the Scality S3 server:
+In a new shell, start the Scality S3 server with the record log enabled:
 
 ```
 git clone https://github.com/scality/s3 ~/replication/s3 && \
 cd ~/replication/s3 && \
 npm i && \
+sed -i '' 's/"enabled": false.*/"enabled": true,/' config.json \
```

Comment:
This is too hacky IMO, I would advise to split this in three steps, with explaining what to change in the config manually in between and an explanation of what this change does to inform the user.

`jq` would be a nice alternative though it messes up the sections and is not part of the bare unix command line tools so has to be installed first, but why not. Also `sed` has a way to define boundaries as regexp where to do the substitutions which would be safer but still hacky.

Reply:
Agree. I will update this to just explain the change manually.